src/app/migration/src/src/app/modules/core/services/user/user.service.ts

```diff
@@ -43,12 +44,28 @@ export class UserService {
    * reference of lerner service.
    */
   public learner: LearnerService;
+  /**
+ * Reference of appId
+ */
+  public _appId: string;
+  /**
+   * Reference of channel
+   */
+  public channel: string;
+  /**
+   * Reference of dims
+   */
+  public dims: Array<string> = [];
+  /**
+   * Reference of Ekstep_env
+   */
+  public _env: string;
```

Comment:
this should be private, get method is created to access this

Reply:
When copying files from other branch to new branch i missed to update appid and channelid, now i added